php-pecl/CartAddModel-oauth1-pecl.php

```diff
@@ -11,18 +11,18 @@
     $oauth->enableDebug();
     $oauth->setToken($access_token, $access_secret);
 } catch(OAuthException $E) {
-    Error("setup exception", $E->getMessage(), null, null, $E->debugInfo);
+    Error("setup exception", $E->getMessage(), null, null, $E->debugInfo, $E->getFile(), $E->getLine());
 }
 
 try {
-    $modelId = 1234567; # CHANGEME
+    $modelId = 24470; # CHANGEME
```

Comment:
its a real model that will successfully add it to their cart - they should still change it but this way it might be easier for some folks to see a successful post